presentation-compiler/test/dotty/tools/pc/tests/completion/CompletionSuite.scala

```diff
@@ -1672,18 +1674,41 @@ class CompletionSuite extends BaseCompletionSuite:
       topLines = Some(5)
     )
 
-
   @Test def `import-rename` =
     check(
       """import scala.collection.{AbstractMap => Set@@}
         |""".stripMargin,
       ""
     )
 
+  @Ignore
```

Comment:
This test was added a week ago and the comment made by Dale makes sense that we should not wrap this into try, as this will only hide the underlying problem.

https://github.com/lampepfl/dotty/pull/19488
I can restore this change in this PR, and then remove this test in the next one.

Reply:
Nah, if that's from like a week ago I think it's fine either way.